src/input.rs

```diff
@@ -10,15 +10,18 @@ use crate::{
     VirtualJoystickType,
 };
 
-#[derive(Event)]
-pub enum InputEvent {
-    StartDrag { id: u64, pos: Vec2 },
-    Dragging { id: u64, pos: Vec2 },
-    EndDrag { id: u64, pos: Vec2 },
+#[derive(Clone, Copy, Debug, Default, Reflect, PartialEq, Eq)]
+pub enum TouchOrMouse {
+    #[default]
+    Touch,
+    Mouse,
 }
 
-pub fn run_if_pc() -> bool {
-    !["android", "ios"].contains(&std::env::consts::OS)
+#[derive(Event)]
+pub enum InputEvent {
+    StartDrag { id: u64, pos: Vec2, touch_or_mouse: TouchOrMouse },
+    Dragging { id: u64, pos: Vec2, touch_or_mouse: TouchOrMouse },
+    EndDrag { id: u64, pos: Vec2, touch_or_mouse: TouchOrMouse },
```

Comment:
In this case that we have just 2 variants it seems better to me that it is a boolean, what do you say? The boolean could be called something like `isTouch` or `isMouse` and change its values depending on the one or the other